offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/ProjectInformationView.groovy

```diff
@@ -73,6 +74,12 @@ class ProjectInformationView extends VerticalLayout {
         this.addComponents(textLayout, buttonLayout)
     }
 
+    private void resetContent() {
+        this.projectTitle.setValue("")
+        this.projectObjective.setValue("")
+        this.experimentalDesign.setValue("")
```

Comment:
by setting these values the placeholders are not set again, right? I think you need to call .clear() or something similar just as in the CreateCustomerView

Reply:
they are displayed